platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/extensions/event/LogMessage.java

```diff
@@ -25,7 +27,7 @@
  * @see SimpleLogMessage
  * @see ParameterizedLogMessage
  */
-public sealed interface LogMessage permits SimpleLogMessage, ParameterizedLogMessage {
+public sealed interface LogMessage permits ParameterizedLogMessage, SimpleLogMessage {
```

Comment:
I asked myself if sealed makes sense here. We should do a short discussion about that